Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Might be good to have an overview of what the steps are going to be. Maybe also say in the requirements which tables need to present - SecurityAlerts, W3CIISLog.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Cheers. I've made the overview clearer now, the requirements at the top of the notebook include the required data sources.</p>